DataFormats/Reconstruction/include/ReconstructionDataFormats/MatchInfoTOF.h

```diff
@@ -25,20 +25,21 @@ namespace dataformats
 {
 class MatchInfoTOF
 {
-  using evGIdx = o2::dataformats::EvIndex<int, o2::dataformats::GlobalTrackID>;
+  //  using evGIdx = o2::dataformats::EvIndex<int, o2::dataformats::GlobalTrackID>;
+  using evGIdx = o2::dataformats::GlobalTrackID;
   using evIdx = o2::dataformats::EvIndex<int, int>;
```

Comment:
the 1st int of EvIndex refers to the "entry in the tree", let's eliminate this also: the MatchInfo will use only GID of the track and int index for clusters (can be also a GID but this will be an overkill, since the clusters may come only from TOF anyway)

Reply:
done

Reply:
Actually, seems it was not done: could you simply remove  `using evIdx = o2::dataformats::EvIndex<int, int>;`
and make the data members
````
int mTOFClRef;
GTrackID mTrackRed;
````

Reply:
Done